website/blog/2024-05-17-whats-next.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: whats-next
+title: What's next for MACI?
+description: Dive into the next MACI protocol changes and improvements.
+authors:
+  name: ctrlc03
+  title: MACI dev
+  url: https://x.com/ctrlc03
+  image_url: https://avatars.githubusercontent.com/u/93448202?v=4
+tags: [voting, security, anonymity]
+---
+
+Minimal Anti Collusion Infrastructure ([MACI](https://github.com/privacy-scaling-explorations/maci)) is a Public Good that allows one to run secure, private, on-chain voting polls.
+
+Given its open source nature, it is not uncommon to develop new features or to have to fix issues based on community feedback. However, something that does not usually happen is external contributors making changes to the core protocol.
+
+Well, this soon will be a reality thanks to a proposal sent by the [3327](https://3327.io/) team. 3327 is collective of 10+ people working on improving blockchain technologies, with a focus on research and engineering.
+
+This proposal can be divided into two parts. First of all, the idea is to bring unconditional privacy to MACI's voters. After which, the focus would shift to optimising inefficient merkle tree structure holding messages, by replacing it with a hash chain.
+
+## Provide unconditional voter privacy
+
+Currently with MACI, if a voter performs a [key change](https://maci.pse.dev/docs/key-change), the voter's new key would not be anonymous. The coordinator could collude with a bad actor to inform the latter of the key change, as the coordinator would have access to all decrypted messages.
+
+The key focus of this improvement would be to allow users to be completely anonymous by removing the link between the original signup key and the key used for voting. How would this work? Well, users sign up to vote via the [MACI contract](https://maci.pse.dev/docs/contracts#macisol), and depending on the gatekeeper in use, they'd have to prove that they've passed the entry condition. Now, given knowledge of this key, they can signup with a new key to polls deployed by this same MACI contract.
+
+Thus, they can prove anonymously that they know the preimage of a [`StateLeaf`](https://maci.pse.dev/docs/typedoc/domainobjs/classes/StateLeaf), by passing this information to a zk-SNARK circuit, and validating this proof within the poll contract when joining with the new key. Of course you might be thinking that everyone knows the preimage of a state leaf, as it's public information that can be taken from the contracts' logs. However, the circuit will not accept the public key directly but would instead take the private key and use it to generate the public key. This way, only users with knowledge of a specific private key can generate a valid inclusion proof.
+
+Now after signing up to the poll with this new key, there will not be any link to the original key, and users will effectively be anonymous. Of course users should ensure that they are using different wallets where possible.
```

Comment:
```suggestion
Now after signing up to the Poll with this new key, there will not be any link to the original key, and users will effectively be anonymous. Of course users should ensure that they are using different wallets where possible.
```
Looks like we capitalize Poll sometimes but not other times - let's check that this is consistent (I suppose I'd capitalize all MACI Polls?)